### DISCUSSION TITLE
question: am i in the latest version?

### DISCUSSION THREAD

**Discussion by jeimz143** [quasarframework/quasar#9831] [General - Components / Directives / etc] [answered]
Hi i just want to confirm, i am trying to install the latest version of quasar and i tried to quasar create and i notice that in my script we have this `export default defineComponent()` in every component. but in the documentation  we only have `export default`.  i just want to confirm if i am pulling the latest version? Thank you.

**Comment by hawkeye64** [comment, marked as answer]
`defineComponent` is Vue 3, but it's a no-op method, but needed for Typescript.

**Reply by jeimz143** [reply]
oks got it.. Thank you!
